Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -161,15 +161,28 @@
 
     <!-- XTXT: risk card - High risk state - Days with high risk encounters -->
     <plurals name="risk_card_high_risk_encounter_days_body">
+        <item quantity="one">"Begegnungen an %1$d Tag mit erhöhtem Risiko"</item>
+        <item quantity="other">"Begegnungen an %1$d Tagen mit erhöhtem Risiko"</item>
```

Comment:
What exactly would be a possible value for "other"?

Reply:
from the [Android docs](https://developer.android.com/guide/topics/resources/string-resource#Plurals)

> When the language does not require special treatment of the given quantity (as with all numbers in Chinese, or 42 in English).

We have all possible quantities in every language. This is actually not necessary - e.g. German only needs `one` and `other` and the app would also work if you only provide these two quantities. 

Android Studio shows me which quantities are relevant for each language, so I could remove the unnecessary ones if you like ...

Reply:
No, at least not for me ;-)

Reply:
That is from the translation system.